src/page-response-impact/ResponseImpactDashboard.tsx

```diff
@@ -135,6 +157,12 @@ const ResponseImpactDashboard: React.FC = () => {
         <ValidDataWrapper facilities={facilities.data}>
           <PageContainer>
             <Column>
+              <Link to="/">
```

Comment:
Question: I wasn't sure where this link should go to?

Reply:
I believe it would link back to the multi-facility page, but you should double check with @sychang.

Reply:
is good for now